File: tests/02_b_1.py
#!/usr/bin/env python3

from interp import *

def t():
    c = [
    ]
    i = [
        cpu_encode_op_b(2),
        cpu_encode_op_or_i64(0,0,0),
        cpu_encode_op_break(0),
    ]
    cpu_test("b_1", c, i)

t()

File: tests/02_b_1.go
package main

import . "glyph/tests/interp"

func t() {
    c := []uint64{
    }
    i := []uint16{
        CPU_encode_op_b(2),
        CPU_encode_op_nop(0),
        CPU_encode_op_break(0),
    }
    CPU_test("b_1", c, i);
}

func main() {
    t()
}
